extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts

```diff
@@ -59,6 +59,7 @@ export class PublishDatabaseDialog {
 	private serverName: string | undefined;
 	protected optionsButton: azdataType.ButtonComponent | undefined;
 	private publishOptionsDialog: PublishOptionsDialog | undefined;
+	public fileUris: vscode.Uri[] | undefined;
```

Comment:
can this have a more descriptive name? When would there be multiple fileUris?

Reply:
This variable is for getting the profile options when profile exists, but we are not at that decision to how we handle the profile, I'm removing this logic, and default options will be the profile options if profile is loaded else deployoptions but reset always be default deployoptions as is till we decide it.